WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignItemView.swift

```diff
@@ -87,7 +87,7 @@ struct BlazeCampaignItemView: View {
                 VStack(alignment: .leading, spacing: Layout.statsVerticalSpacing) {
                     Text(Localization.budget)
                         .subheadlineStyle()
-                    Text(String(format: "%.2f", campaign.totalBudget))
+                    Text(String(format: "%.0f", campaign.budgetCents/Constants.centsToUnit))
```

Comment:
```suggestion
                    Text(String(format: "%.0f", campaign.budgetCents / Constants.centsToUnit))
```

Reply:
Updated in e6a6617dac9c66ffdfc6ff93ea4ea72c603f91b7.